src/cript/api/api.py

```diff
@@ -194,7 +191,11 @@ def _get_s3_client(self) -> boto3.client:  # type: ignore
 
         identity_id = auth.get_id(IdentityPoolId=self._IDENTITY_POOL_ID, Logins={self._COGNITO_LOGIN_PROVIDER: self._token})
 
-        aws_credentials = auth.get_credentials_for_identity(IdentityId=identity_id["IdentityId"], Logins={self._COGNITO_LOGIN_PROVIDER: self._token})
+        aws_token = self._token
+        if len(aws_token.split()) > 1:
+            # Remove ID prefix like 'Bearer'
+            aws_token = aws_token.split()[-1]
```

Comment:
```suggestion
aws_token = aws_token.lstrip("Bearer ")
```

Reply:
please consider changing this one this is easy and will make the code cleaner, I know that it's temporary but lets make our code clean anyways

Reply:
sure, I add it.